CQA DOCPLAN-175 flatten toc (#110808)
openshift/openshift-docs: virt/managing_vms/virt-accessing-vm-consoles.adoc

[role="_abstract"]
By using VNC, serial, or desktop viewer consoles, you can access the console of your virtual machine for troubleshooting when the VM does not have network connectivity.

> include::modules/virt-vm-console-access-considerations.adoc[leveloffset=+1]

Reviewer: ```suggestion
include::modules/<the_new_Considerations_for _accessing_the_VNC_console_module>.adoc[leveloffset=+1]
```

> include::modules/virt-vm-console-access-considerations.adoc[leveloffset=+1]

Reviewer: There must an empty line after every include statement

> include::modules/virt-connecting-vnc-console-web.adoc[leveloffset=+1]

Reviewer: ```suggestion
include::modules/virt-connecting-vnc-console-web.adoc[leveloffset=+2]
```

Author: Need more input or advice on the leveloffset recommendations. The flattening here is to satisfy the following: https://redhat.atlassian.net/wiki/spaces/cnv/pages/268599829/openshift-docs+virt+managing_vms+-+CQA+March+2026

"Requirement (CQA Pre-migration): Content is not deeply nested in the TOC (recommended: no more than 3 levels)."

All of these modules are nested under 9.5, any level beyond =+1 would be 4 levels deep.

Reviewer: So I think my thought process here was to group the VNC-related modules under the "Considerations for accessing the VNC console" and the serial-related modules under the "Considerations for accessing the serial console". 

I am okay with keeping the flat structure (the way you have it right now) if grouping them violates the TOC nesting requirement for DITA.

> include::modules/virt-connecting-vnc-console-virtctl.adoc[leveloffset=+1]

Reviewer: ```suggestion
include::modules/virt-connecting-vnc-console-virtctl.adoc[leveloffset=+2]
```

include::modules/virt-generate-temporary-token-VNC.adoc[leveloffset=+1]

> include::modules/virt-grant-token-generation-permission-VNC.adoc[leveloffset=+1]

Reviewer: The order of the modules does not make sense. Shouldn't granting the token permissions come first, then generating the token and finally connecting to the VNC console?

Reviewer: I still have the same question about the order of the modules.

Author: Sorry for missing this — fixed!

> include::modules/virt-connecting-serial-console-web.adoc[leveloffset=+1]

Reviewer: If you go with my earlier suggestion, there would be an include statement to add the "Considerations for accessing the serial console" module with leveloffset=+1 before this

```suggestion
include::modules/virt-connecting-serial-console-web.adoc[leveloffset=+2]
```

> include::modules/virt-connecting-serial-console-virtctl.adoc[leveloffset=+1]

Reviewer: ```suggestion
include::modules/virt-connecting-serial-console-virtctl.adoc[leveloffset=+2]
```

include::modules/virt-connecting-desktop-viewer-web.adoc[leveloffset=+1]

[role="_additional-resources"]

> .Additional resources

Reviewer: ```suggestion
[id="additional-resources_{context}"]
== Additional resources
```

Author: resolved
